### PR TITLE
#258 add example section in default template

### PR DESCRIPTION
**Pull request by ansyral**
@vicancy @superyyrrzz @vwxyzh

### CODE REVIEW

File: src/docfx.website.themes/default/partials/class.tmpl.partial [hunk 1]
```diff
@@ -87,6 +87,12 @@
 <h5 id="{{id}}_remarks"><strong>{{__global.remarks}}</strong></h5>
 <div class="markdown level0 remarks">{{{remarks}}}</div>
 {{/remarks}}
+{{#example.0}}
+<h5 id="{{id}}_examples"><strong>{{__global.examples}}</strong></h5>
+{{#example}}
+{{{.}}}
```
Comment (vicancy) [inline src/docfx.website.themes/default/partials/class.tmpl.partial]: what does `{{{.}}}` mean?

Comment (ansyral) [inline src/docfx.website.themes/default/partials/class.tmpl.partial]: example is a list of string, and {{{.}}} would take the content of a string

**Comment by vwxyzh** [comment]
👍

**Comment by vicancy** [comment]
:+1:  LGTM

**Pull request merged**